tests/test_views_rest.py

```diff
@@ -27,15 +27,439 @@
 
 from __future__ import absolute_import, print_function
 
-import hashlib
 import json
 from time import sleep
 
 from flask import url_for
 from flask_security import url_for_security
 from six import BytesIO
 
-from invenio_deposit.api import Deposit
+
+def test_edit_deposit_1(app, db, es, users, location, deposit, write_token):
+    """Test edit deposit by the owner."""
+    deposit_id = deposit['_deposit']['id']
+    with app.test_request_context():
+        with app.test_client() as client:
+            # login
+            res = client.post(url_for_security('login'), data=dict(
+                email=users[0].email,
+                password="tester"
+            ))
+            res = client.put(
+                url_for('invenio_deposit_rest.dep_item', pid_value=deposit_id),
+                data=json.dumps({"title": "bar"}),
+                headers=[('Content-Type', 'application/json'),
```

Comment:
It would be nice to have fixture for `header` and `auth_headers` that you are using everywhere.